controllers/mealController.js

```diff
@@ -7,11 +7,12 @@ const meals = { 1:"Breakfast", 2:"Snack", 3:"Lunch", 4:"Dinner" }
 exports.meals = (request, response) => {
   Meal.all()
     .then((meals) => {
-      // Promise.all([meals.forEach(meal => Meal.foodsForMeal(meal))])
-      //   .then((foods) => {
-      //     eval(pry.it)
-      //   })
-
+      // Promise.all([
```

Comment:
are you still working on this?

Reply:
Yup. I have to create a meal object and pass in foods as an array.